backend/routes/follows.js

```diff
@@ -42,7 +42,7 @@ router.post("/follow", async (req, res) => {
     const user = req.session.user;
     const ticker = req.body.ticker?.toUpperCase();
 
-    if (!ticker) {
+    if (ticker == null) {
```

Comment:
I like the added explicit null checking here

Reply:
Me too :D